src/cli.ts

```diff
@@ -24,12 +24,14 @@ interface CLIOptions {
   paths: Array<string>;
   baseOptions: Options;
   modernizeJS: boolean;
+  output: string;
```

Comment:
To follow the advice I gave about making `output` be potentially `undefined`, this should be `output?: string;`.